code/modules/hallucinations/hallucinations.dm

```diff
@@ -56,7 +56,7 @@ GLOBAL_LIST_INIT(hallucinations, list(
 	switch(rand(100))
 		if(0 to HALLUCINATE_MINOR_WEIGHT)
 			severity = HALLUCINATE_MINOR
-		if((HALLUCINATE_MINOR_WEIGHT + 1) to HALLUCINATE_MODERATE_WEIGHT)
+		if((HALLUCINATE_MINOR_WEIGHT + 1) to (HALLUCINATE_MINOR_WEIGHT + HALLUCINATE_MODERATE_WEIGHT))
```

Comment:
This should fix moderate hallucinations not occurring. Unsure if this happens on live.
If so, can you add this to the CL?

Reply:
Yeah, will add it